Automattic/wp-calypso
# Update/backup view for wordpress com

#### Changes proposed in this Pull Request

* add `jetpack/backups-date-picker` feature flag to control complexity of backup picker
* activate for green calypso

![Untitled 9@0 5x](https://user-images.githubusercontent.com/2810519/84936715-8831d300-b08f-11ea-978f-372011f671cf.png)


#### Testing instructions

1. Boot Blue Calypso, verify that the calendar and search icon are hidden on the backup page
2. Boot Green Calypso, verify that the calendar and search icon appear on the backup page
